chart/env/prod.yaml

```diff
@@ -106,6 +106,8 @@ optInOutUrlsScan:
   rowsMaxNumber: 100_000
   urlsNumberPerBatch: 1000
 
+descriptiveStats:
+  maxParquetSizeBytes: "1_000_000_000"
```

Comment:
I actually just aligned the value with duckdb-index, for some reason I thought that it also uses 1Gb. But yes, I think here it can be increased to 5G.